docs/getting-started.md

```diff
@@ -28,9 +42,6 @@ $ SCALARDB_HOME=/path/to/scalardb
 $ cd $SCALARDB_HOME
 $ ./gradlew installDist
 $ sudo mkdir /var/log/scalar; sudo chmod 777 /var/log/scalar
```

Comment:
```suggestion
$ sudo mkdir /var/log/scalar;
$ sudo chmod 777 /var/log/scalar
```

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.